test/functional/services/find.js

```diff
@@ -6,11 +6,12 @@ export function FindProvider({ getService }) {
   const defaultFindTimeout = config.get('timeouts.find');
 
   class Find {
-    byCssSelector(selector) {
+    async byCssSelector(selector, timeout = defaultFindTimeout) {
       log.debug(`findByCssSelector ${selector}`);
-      return remote
```

Comment:
I think a helper method would be useful here. Maybe something like:

```js
class Find {
  async withTimeout(timeout, block) {
    try {
      const remoteWithTimeout = remote.setFindTimeout(timeout);
      return await block(remoteWithTimeout);
    } finally {
      remote.setFindTimeout(defaultFindTimeout);
    }
  }
}
```

Then these methods functions could be more focused on their specific details:

```js
class Find {
  async byCssSelector(selector, timeout = defaultFindTimeout) {
    return await this.withTimeout(timeout, async remote => {
      return await remote.findByCssSelector(selector);
    })
  }
}
```